.vscode/tasks.json

```diff
@@ -5,20 +5,40 @@
   "tasks": [
     {
         "label": "build",
-        "command": "./build.cmd",
-        "type": "shell",
+        "command": "dotnet",
         "args": [
+          "msbuild",
+          "-p:GenerateFullPaths=true",
+          "Compilers.sln"
         ],
+        "windows": {
```

Comment:
I get errors when trying to build roslyn in windows using this task. consider invoking `./build.sh` or `./build.cmd` depending on platform.

I get several errors that look like this: `C:\Program Files\dotnet\sdk\3.1.300-preview-015095\Microsoft.Common.CurrentVersion.targets(3032,5): error MSB3823: Non-string resources require the property GenerateResourceUsePreserializedResources to be set to true. [C:\Users\rikki\src\roslyn\src\VisualStudio\VisualStudioDiagnosticsToolWindow\Roslyn.VisualStudio.DiagnosticsWindow.csproj]`